ansible/roles/software/ckan/catalog/ckan-app/tasks/worker.yml

```diff
@@ -28,6 +28,18 @@
     - usr/bin/celery-mem-check.sh
   when: catalog_ckan_worker_type == 'qa_worker'
 
+- name: schedule ckan-clean-harvest-logs cron
+  cron:
+    name: "clean harvest logs in db"
+    cron_file: ckan
```

Comment:
You'll have to give this file a different name. The next task will overwrite /etc/cron.d/ckan and this change will be lost.

FYI, if we had idempotency checks, this would've been caught by molecule.